Get-Started.md

```diff
@@ -0,0 +1,111 @@
+- For any given input it returns a string of bits of the same size.
+- Every object has one cryptographic hash.
+- It is impossible (well, really really difficult) given a hash to know what content it refers to 
+- and given two hashes to know if the content was similar.
+
+- The hash of object1 is XXXX (well, this is a multihash, read more here)
+- if we remove the dot from Mr, the hash is YYYY.
+- In IPLD cryptographic hashes ensure that the content cannot change.
+- If we ask to a network (like the ipfs network) for a hash
+- then, they cannot lie about the content that they are giving
+- since you can generate the hash of the content received and check if it is the same one to the one that you asked for!
+
+EXERCISE: Write your own IPLD object and use the IPFS command line to add it to the network
+
+EXERCISE: Show how to do that in code too
+
+### Path name into one file
+
+- IPLD offers a path scheme that allow you to link inside an IPLD object (and as we will see later, across IPLD objects)
+- Every key is like a folder.
+- We can explore the previous object in this way:
+- XXXXX/name points to: “Mr. Slippery”
+- XXXXX/job/0: writer
+
+## Linking objects
+
+- What about secure linking?
+- Differently from standard JSON, we can link two objects by using a special keyword: “/“
+- For example:
+
+```
+object1 = {
+  name: “Mailman”,
+}
+
+TODO hash of object1
+
+object2 = {
+  name: “Mr. Slippery”,
+  job: [“Writer”, “Spy”],
+  friends: [{“/”: “YYYY”}]
+}
+
+TODO: hash of object2
+
+
+```
+
+- Links are cryptographic hashes, we often call those merkle links (in reference to Merkle Tree)
+- Since we are using cryptographic hashes, then the links can only be one way (link to stackoverlow issue that mentions this)
+- The IPLD path scheme can go across objects. For example
+- YYYY/friends/0/name would point to Mailman!
+
+EXERCISE: Show How to link objects in JS
+
+## Merkle Dag
+
+- If you have many objects linked to each other, then you are building a Merkle DAG that could look exactly like this
+- TODO: Show a big Merkle DAG
+- The great feature about IPLD and Merkle Dags is that you only need the hash 
+- of the root of the dag to authenticate (“validate the correctness”) of the whole Merkle Dag!
+
+# Dive deep into IPLD
+- Let's move beyond toy examples and let's learn more about IPLD
+## Path scheme
+- An IPLD address is composed by a CID and a path.
+- The CID is a content identifier that is composed of X,Y,Z.
+- A multihash is not just a hash: it is the hash prefixed by the hashing function and the length of the hash. In this way we can decide to what cryptographic hash function we want to hash the content.
+- A multicodec specifies the type of content we are addressing.
+- In the examples above we used IPLD+CBOR
+- Thanks for recent work in IPLD, we can address natively other type of content. For example, a format we recognize is an Ethereum block. So we can now content address an ethereum block and use our path scheme on it!
```

Comment:
Thanks to recent work in IPLD, we can now address other types of content natively.